test/cloud_testing/platforms/slc6_x86_64_test.sh

```diff
@@ -73,12 +73,68 @@ CVMFS_TEST_CLASS_NAME=ServerIntegrationTests                                  \
                               -x src/518-hardlinkstresstest                   \
                                  src/524-corruptmanifestfailover              \
                                  src/585-xattrs                               \
+                                 src/700-overlayfsvalidation                  \
                                  --                                           \
                                  src/5*                                       \
                                  src/6*                                       \
+                                 src/7*                                       \
                               || retval=1
 
 
+echo -n "starting FakeS3 service... "
```

Comment:
The S3 tests unfortunately need to be disabled for the time being.  They all fail, I think because of FakeS3.  But we didn't look into it yet.